Installers/Ubuntu-Install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+echo Please enter your password to continue...
+
+# Install the following dependencies from repositories without asking questions, git-core can be removed afterwords as it's only used to initialise a local git repository
+echo Installing dependencies
+sudo apt-get -y install git-core libusb-1.0.0 libudev-dev python3 cython3 python3-pip python3-pil python3-setuptools
+
+cd /tmp
+pip3 install hidapi click
+
+# Get the sudoers home directory before changing directory into it
+cd ~
+
+# If the git repository already exists then delete it
+if [ -d "python-evic" ]; then
+  rm -Rf python-evic
```

Comment:
I really don't like removing directories from users home. Maybe create a temp directory for cloning?